client/main.lua

```diff
@@ -27,32 +27,32 @@ local function verifyTypes(notiTable, isPersistent)
     return false
   end
 
-  if not notiTable.style or type(notiTable.style) ~= 'string' then
+  if type(notiTable?.style) ~= 'string' then
```

Comment:
This should now support optional chaining instead of checking to see if it exists.

Reply:
This has now been changed as its purpose was modified from its original intents.